pkgs/by-name/co/cook-cli/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildNpmPackage,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  darwin,
```

Would be a nicety if you allow building the GUI to be opt-outtable
```suggestion
  darwin,
  withGui = true,
```

---

Unfortunately the rust project doesn't have feature flags, so the UI (it's a local web server) is a requirement to build the CLI.

---

Oh, that's weird. No worries, then.